connect/src/whscan-api.ts

```diff
@@ -262,11 +263,16 @@ export async function getRelayStatusWithRetry(
   return retry<RelayData>(task, WHSCAN_RETRY_INTERVAL, timeout, "Wormholescan:GetRelayStatus");
 }
 
-export async function getVaaByTxHash(rpcUrl: string, txid: string): Promise<ApiVaa | null> {
+export async function getVaaByTxHash(rpcUrl: string, txid: TransactionId): Promise<ApiVaa | null> {
   const url = `${rpcUrl}/api/v1/vaas?txHash=${txid}`;
```

Comment:
is there a way to specify the chain id in this query instead of filtering after receiving the response? not sure if that would also be more efficient on the server side.

Reply:
It doesn't look like it unfortunately https://wormholescan.io/#/developers/wormholescan-doc/api-doc/get-vaas